.github/workflows/rosa-multi-az-clusters_single-cluster-run-benchmark.yml

```diff
@@ -0,0 +1,368 @@
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '21'
+          cache: 'maven'
+
+      - name: Cache Maven Wrapper
+        uses: actions/cache@v4
+        with:
+          path: |
+            .mvn/wrapper/maven-wrapper.jar
+          key: ${{ runner.os }}-maven-wrapper-${{ hashFiles('**/maven-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-wrapper-
+
+      - name: Build with Maven
+        run: |
+          ./mvnw -B clean package -DskipTests -pl benchmark
+          tar xfvz benchmark/target/keycloak-benchmark-*.tar.gz
+          mv keycloak-benchmark-* keycloak-benchmark
+
+      - name: Get URLs
+        uses: ./.github/actions/get-keycloak-url
+        with:
+          project: ${{ env.PROJECT }}
+
+      - name: Create Keycloak dataset with "${{ inputs.usersPerRealm }}" users and "${{ env.CLIENTS_PER_REALM }}" clients
+        if: ${{ !inputs.skipCreateDataset }}
+        uses: ./.github/actions/keycloak-create-dataset
+        with:
+          project: ${{ env.PROJECT }}
+          users: ${{ inputs.usersPerRealm }}
+          clients: 100
+          clientsPerRealm: ${{ env.CLIENTS_PER_REALM }}
+          createClientForSpecificRealm: true
+          maxWaitEntityCreation: ${{ inputs.maxWaitEntityCreation }}
+
+      - name: Restart Keycloak Pods after dataset creation
+        uses: ./.github/actions/keycloak-restart-pods
+        with:
+          project: ${{ env.PROJECT }}
+          keycloak_operator_name: ${{ inputs.keycloak_operator_name }}
+
+      - name: Create AWS EC2 instances
+        id: create_aws_ec2_instances
+        uses: ./.github/actions/ec2-create-instances
+        with:
+          region: ${{ inputs.region }}
+
+      - name: Prepare horreum report
+        uses: ./.github/actions/prepare-horreum-report
+        with:
+          createReportFile: true
+
+
+      - name: Run Memory Usage Total Query Before Benchmark on Cluster 1
+        uses: ./.github/actions/prometheus-run-queries
+        with:
+          project: ${{ env.PROJECT }}
+          runMemoryUsageTotal: true
+          output: memory_create_sessions
+      - name: Testing memory for creating sessions
+        id: kcb-authorization-code-1
+        run: >
+          ./benchmark.sh ${{ inputs.region }}
+          --scenario=keycloak.scenario.authentication.AuthorizationCode
+          --server-url=${{ env.KEYCLOAK_URL }}
+          --realm-name=realm-0
+          --users-per-sec=${{ inputs.numberOfUsersPerSecond }}
+          --ramp-up=20
+          --logout-percentage=0
+          --measurement=${{ inputs.measurement }}
+          --users-per-realm=${{ inputs.usersPerRealm }}
+          --clients-per-realm=${{ env.CLIENTS_PER_REALM }}
+          --log-http-on-failure
+          --sla-error-percentage=0.001
+        continue-on-error: true
+        working-directory: ansible
+      - name: Run Memory Usage Total Query After Benchmark on Cluster 1
```

Comment:
It would be good to cleanup the various workflow files so that we have one empty line between steps, it makes it much easier to read.